### PR TITLE
[doc] Update to include API for checking JVM pointers

### PR DESCRIPTION
**Pull request by a03nikki**
Add information for how to find if compressed ordinary object pointers is in use using the REST APIs.

Currently the documentation only includes how to check it via the log file.
https://www.elastic.co/guide/en/elasticsearch/reference/8.1/advanced-configuration.html#set-jvm-heap-size
https://www.elastic.co/guide/en/elasticsearch/reference/7.17/advanced-configuration.html#set-jvm-heap-size

This documentation update should be back ported to at least the 7.17 version as I tested the command against v7.17.

### CODE REVIEW

**Comment by a03nikki** [comment]
> @a03nikki please enable the option "Allow edits and access to secrets by maintainers" on your PR. For more information, [see the documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

I think this message is because I poorly named the branch. 🤦  I won't use a `/` in my future branch names.

File: docs/reference/setup/advanced-configuration.asciidoc
Comment (a03nikki) [inline docs/reference/setup/advanced-configuration.asciidoc:121]: What is the correct method for referring to a field returned in the JSON response?
I didn't know so I put in `{name}` as I wasn't sure what else to call that level.

```json
{
  "_nodes" : {
    "total" : 1,
    "successful" : 1,
    "failed" : 0
  },
  "cluster_name" : "...",
  "nodes" : {
    "iIo93x-eSIyrvodrPuOOHw" : {
      "name" : "instance-0000000016",
...
      "version" : "7.17.3",
...
      "roles" : [...],
      "attributes" : {...},
      "jvm" : {
...
        "mem" : {...},
        "gc_collectors" : [...],
        "memory_pools" : [...],
        "using_compressed_ordinary_object_pointers" : "true",
        "input_arguments" : [...]
      }
    }
  }
}
```

Comment (a03nikki) [reply docs/reference/setup/advanced-configuration.asciidoc:121]: At minimum, I should probably update it to be `{id}` instead of `{name}`.
```suggestion
Or check the <<cluster-nodes-info,nodes info API's>> `nodes.{id}.jvm.using_compressed_ordinary_object_pointers` value:
```

Comment (grcevski) [inline docs/reference/setup/advanced-configuration.asciidoc:121]: I wonder if you can say `jvm.using_compressed_ordinary_object_pointers` under the specific node id. Curly braces have special meaning I think.

Comment (a03nikki) [reply docs/reference/setup/advanced-configuration.asciidoc:121]:
```suggestion
Or check the `jvm.using_compressed_ordinary_object_pointers` value for the nodes using the <<cluster-nodes-info,nodes info API>>:
```

**Comment by a03nikki** [comment]
I checked the build and the page looks to format correctly.
https://elasticsearch_86360.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/advanced-configuration.html

**Review by grcevski — APPROVED** [review]
LGTM! Thanks!

**Comment by a03nikki** [comment]
@grcevski : What version tags should I have on this PR before merging?  I would _like_ the change to be merged to the 7.17 (last in 7.x), 8.2 (current), and future versions on our website.

**Comment by grcevski** [comment]
I added them for you. So when you 'squash and merge' now with the auto-backport-and-merge label, the CI will create separate PRs for each backport and if tests pass it will merge the changes automatically. If there are conflicts during the merge on any of the PRs, you'll need to check out the backport branch and resolve the conflicts manually. You don't need another review to merge those backport PRs (unless there are non trivial changes), just use the admin privilege to do the merges. I also don't mind reviewing any of the backports if you'd like me to :).

**Comment by a03nikki** [comment]
@grcevski : (1) How do I resolve the backport failures?  (2) Do I just create manual PRs for the 8.1 and 8.2 branches? 

> The backport operation could not be completed due to the following error: 		
> 
> ```
> An unhandled error occurred. Please consult the logs
> ```
> 
> You can use [sqren/backport](https://github.com/sqren/backport) to manually backport by running `backport --upstream elastic/elasticsearch --pr 86360`

(3) Also should this get merged into the 8.0 as well as we did the other two published 8.x releases?

**Comment by grcevski** [comment]
Hi @a03nikki, it looks like the 8.1 and 8.2 backports worked great, PRs were created and auto merged, so nothing left to do there.

7.17 backport didn't work for some reason, the PR was created here https://github.com/elastic/elasticsearch/pull/86485, but it didn't auto-merge. It looks like you already resolved this, so that's all great.

I don't think we need to worry about 8.0, there's no further 8.0.x release planned since we've released 2 minor updates since: 8.1 and 8.2.

**Comment by grcevski** [comment]
I'm sorry, I'm wrong for some reason 8.1 and 8.2 PRs didn't get auto created. I don't know why. We'll need to manually create PR for 8.1 and 8.2 branches, just follow the same structure as what was done for 7.17.

**Comment by a03nikki** [comment]
I started fixing the two failing back ports:

* 8.1 : https://github.com/elastic/elasticsearch/pull/86585
* 8.2: https://github.com/elastic/elasticsearch/pull/86586

**Pull request merged**
